### DISCUSSION TITLE
"Analysis, visualization, and integration of spatial datasets" vignette, which data set to download

### DISCUSSION THREAD

**Discussion by philipspear7** [satijalab/seurat#8236] [Q&A]
"Here, we will be using a recently released dataset of sagital mouse brain slices generated using the Visium v1 chemistry. There are two serial anterior sections, and two (matched) serial posterior sections.

You can download the data [here](https://support.10xgenomics.com/spatial-gene-expression/datasets), and load it into Seurat using the [Load10X_Spatial()](https://satijalab.org/seurat/reference/load10x_spatial) function"

The link goes to 10x datasets unfiltered. Searching for "mouse brain sagittal" shows 16 datasets, 10 of which are V1 chemistry. There are two sets of mouse serial sections 1 and 2 with anterior and posterior matched.

I'm proceeding with just one of the sections due to space, but it would be nice to know exactly which dataset was used in the figures so I can be sure my analysis results match the figures and I'm doing everything correctly.

**Comment by GCAT01** [comment]
I started going through this vignette today and had the same question. If you read a bit further in the vignette, it says the dataset can be loaded from their package
`InstallData("stxBrain")
brain <- LoadData("stxBrain", type = "anterior1")`
I did this and haven't had any issues yet although I am not very far in.

**Comment by RenhaoL** [comment]
Just a follow-up to this. I used `brain <- LoadData("stxBrain", type = "anterior1")` but got the error: 

`Error in `[<-.data.frame`(`*tmp*`, , i, value = new("Assay5", layers = list( : 
  replacement has 31053 rows, data has 2696`

Any ideas how to fix this? Thanks in advance!
